content/plugins/pof-importer/pof-importer.php

```diff
@@ -68,27 +126,30 @@ public static function activate() {
         wp_schedule_event( time(), 'hourly', 'daily_cron' );
     }
     public static function deactivate() {
-        wp_clear_scheduled_hook('daily_cron');
+        wp_clear_scheduled_hook( 'daily_cron' );
```

Comment:
The name of the cron hook is not defined in this branch, but should the hook be more specific about the use case? 'daily_cron' is a pretty general name. Could something else be hooked to this hook and thus it might be cleared here also? What if this would be called `pof_importer_daily_cron` instead? No need to change this now, but consider changing this in future branches.